lib/matplotlib/figure.py

```diff
@@ -2239,6 +2239,20 @@ def __init__(self, parent, subplotspec, *,
         self._set_artist_props(self.patch)
         self.patch.set_antialiased(False)
 
+    def get_figure(self, root=None):
```

Comment:
I did not include a docstring here because this method is not included in [figure_api.rst](https://github.com/matplotlib/matplotlib/blob/main/doc/api/figure_api.rst)

Reply:
Not sure whether not listing in the docs is an oversight. But even if it's for internal use only, a docstring is helpful.

If you don't do anything, this inherits the docstring from Artist. If that is the intended behavior, we mark this with the comment `# docstring inherited`. If you'd like a different phrasing for subfigures, please add an explicit docstring.

Reply:
Actually I guess it would also still show for any user that does something like `sfig.getfigure?` in ipython.  Currently it will need a different docstring for the deprecated behaviour.